providers/mongodb.js

```diff
@@ -11,7 +11,7 @@ module.exports = class extends Provider {
 
     async init() {
         const mongoClient = await Mongo.connect(process.env.MONGO_DB_URL, { useNewUrlParser: true });
-        this.db = mongoClient.db(process.env.MONGO_DB_URL.split('/').slice(-1)[0]);
+        this.db = mongoClient.db(process.env.MONGO_DB_URL.split('').slice(-1)[0]);
```

Comment:
Wait, but this is to get the database from the URL and the database is the part after the `/` , wo why would you not split by "/"

Reply:
honestly idk. but after updating a few packages i was forced to remove it for it to work